src/tox_ini_fmt/__main__.py

```diff
@@ -12,6 +15,17 @@
 RED = "\u001b[31m"
 RESET = "\u001b[0m"
 
+CRLF = b"\r\n"
+LF = b"\n"
+CR = b"\r"
+# Prefer LF to CRLF to CR, but detect CRLF before LF
+ALL_ENDINGS = (CR, CRLF, LF)
```

Comment:
There seems to be no need to be global, should be a variable inside detect_line_ending